package io.github.caique.exception;

public class PedidoNaoEncontradoException extends RuntimeException {
    public PedidoNaoEncontradoException() {
        super("Pedido n√£o encontrado. ");
    }
}
